app/views/group_offers/_form.html.slim

```diff
@@ -26,17 +26,7 @@
 
   .row
     .col-xs-12.col-md-4
-      = f.input :necessary_volunteers, collection: 1..10
-    .col-xs-12.col-md-8
-      - if policy(GroupOffer).supervisor_privileges?
-        = f.association :volunteers
-      - else
-        = f.association :volunteers,
-          collection: Volunteer.without_active_assignment.not_in_any_group_offer.not_responsible
-
-  .row
-    .col-xs-12.col-md-4
-      - if policy(GroupOffer).supervisor_privileges?
+      - if policy(GroupOffer).superadmin_or_department_manager?
```

Comment:
What should be checked here is: 
superadmin can select from all volunteers, department manager from the ones available for assignment/group offer.

Reply:
just if i get it correct, but the changes i did in the group offer policy were correct - or not?

Reply:
The policy needs to be adapted, supervisor & dep manager see different collections